Directory.Build.props

```diff
@@ -2,6 +2,7 @@
 
     <PropertyGroup>
         <LangVersion>latest</LangVersion>
+        <Nullable>enable</Nullable>
```

Comment:
I would recommend adding `<TreatWarningAsError>nullable</TreatWarningAsError>` as well

Reply:
https://github.com/AvaloniaUI/Avalonia/blob/master/build/NullableEnable.props

Reply:
I've enabled nullable warnings unconditionally (XamlX doesn't use any of the not-correctly-annotated APIs from netstandard2.0). The current PR has 0 warnings.